stackit/internal/services/iaas/publicip/resource_test.go

```diff
@@ -26,14 +26,16 @@ func TestMapFields(t *testing.T) {
 				PublicIpId: types.StringValue("pipid"),
 			},
 			&iaasalpha.PublicIp{
-				Id: utils.Ptr("pipid"),
+				Id:               utils.Ptr("pipid"),
```

Comment:
can you add a test case where the API response doesn't have the field set at all? Then in the `Model` we should still have a `types.StringNull()`

Reply:
Added a new test case `network_interface_id_nil`